packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -1,8 +1,12 @@
 import type { ReactElement } from 'react';
+import styled from 'styled-components';
```

Comment:
This import won't fail since our storybook babel is configured to read this as `styled-components/native` but this would fail where people have not configured it. But that's okay since this is the exact same case in `BladeProvider` as well. We'll just have to ensure people configure babel correctly.